### PR TITLE
Cmsdb

### PR DESCRIPTION
**Pull request by Benson-J**
Created dashboard & individual edit pages, with links between them.  Only files that need checking are admin.php, and the edit[...].php files.

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin</title>
+</head>
+<body>
+    <a href="editHome.php">Home</a><br>
```
Comment (imogenhallett) [inline admin.php]: Lines 11 - 15 consider adding title and target attributes to your links

File: editAbout.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
+</head>
+<body>
+    <a href="admin.php">Back</a>
```
Comment (imogenhallett) [inline editAbout.php]: consider adding title and target attributes to your link tag

File: editContact.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
+</head>
+<body>
+    <a href="admin.php">Back</a>
```
Comment (imogenhallett) [inline editContact.php]: consider adding title and target attributes to your link tag

File: editHome.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
+</head>
+<body>
+    <a href="admin.php">Back</a>
```
Comment (imogenhallett) [inline editHome.php]: consider adding title and target attributes to your link tag

File: editPortfolio.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
+</head>
+<body>
+    <a href="admin.php">Back</a>
```
Comment (imogenhallett) [inline editPortfolio.php]: consider adding title and target attributes to your link tag

File: editSocial.php [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
+</head>
+<body>
+    <a href="admin.php">Back</a>
```
Comment (imogenhallett) [inline editSocial.php]: consider adding title and target attributes to your link tag

File: editAbout.php [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
```
Comment (imogenhallett) [inline editAbout.php]: consider customising the title tag contents - this is not the admin homepage

File: editContact.php [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
```
Comment (imogenhallett) [inline editContact.php]: consider customising the title tag contents - this is not the admin homepage

File: editHome.php [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
```
Comment (imogenhallett) [inline editHome.php:8]: consider customising the title tag contents - this is not the admin homepage

File: editPortfolio.php [hunk 10]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
```
Comment (imogenhallett) [inline editPortfolio.php]: consider customising the title tag contents - this is not the admin homepage

File: editSocial.php [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin/Home</title>
```
Comment (imogenhallett) [inline editSocial.php]: consider customising the title tag contents - this is not the admin homepage

**Review by imogenhallett — COMMENTED** [review]
The same two comments for each page :-)

**Review by imogenhallett — COMMENTED** [review]
All okay :-)

File: about.html [hunk 12]
```diff
@@ -10,7 +10,7 @@
 </head>
 <body>
     <header>
-        <a href="home.html" class="homeButton">James</a>
+        <a href="." class="homeButton">James</a>
```
Comment (mporam) [inline about.html]: why?

File: admin.php [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | Admin</title>
+</head>
+<body>
+    <a href="editHome.php" title="Edit Homepage">Home</a><br>
```
Comment (mporam) [inline admin.php]: put them in divs instead of using br tags

**Review by mporam — CHANGES_REQUESTED** [review]

File: about.html (hunk 12 above)
Comment (Benson-J) [reply about.html]: Intended to link to index.html, have changed to "index.html"

File: admin.php (hunk 13 above)
Comment (Benson-J) [reply admin.php]: Done

**Review by mporam — APPROVED** [review]

**Pull request merged**